bangazonapi/views/order.py

```diff
@@ -29,11 +30,23 @@ class OrderSerializer(serializers.HyperlinkedModelSerializer):
     """JSON serializer for customer orders"""
 
     lineitems = OrderLineItemSerializer(many=True)
+    total_cost = serializers.DecimalField(
```

Comment:
I noticed is that this total_cost line that was added to the Order serializer doesn't seem to be changing the JSON response for me. I think this is fine since you're handling the total_cost calculations on the client side. I would just suggest taking out line 33-34 to avoid future complications.